### ISSUE TITLE
Deployment of `examples/with-svelte` on Vercel fails

### ISSUE THREAD

**Issue by ogtfaber** [vercel/turbo#8746]
### What is the improvement or update you wish to see?

The example package `examples/with-svelte` currently does not deploy out of the box to Vercel. The issue that shows up is:

`Error: No Output Directory named "public" found after the Build completed. You can configure the Output Directory in your Project Settings.`

Reproduction:
1. Visit https://vercel.com/templates/svelte/turborepo-sveltekit-starter
2. Hit "Deploy" and fill in the fields
3. The issue will show up during the build

### Is there any context that might help us understand?

I have encountered this error moving my project into a monorepo and deploying; I have explored all kinds of issues that have been reported prior, but can't find a resolution.

**Comment by yellowby** [comment]
I have the same issue, but after upgrading to Turborepo 2. Still haven't figured out how to configure build settings on Vercel. Everything was fine Turborepo 1.

**Comment by ogtfaber** [comment]
> I have the same issue, but after upgrading to Turborepo 2. Still haven't figured out how to configure build settings on Vercel. Everything was fine Turborepo 1. 

Interesting. I just got started with Turbo; from what I can find is that folks got it working on Vercel without having to change the output directory manually, even though `public` isn't the most logical name for a Sveltekit app. 

I also filed a ticket, will keep this thread posted when I hear more.

**Comment by ogtfaber** [comment]
I heard back, apparently this issue is fixed in Turbo `2.0.8`. 

Some more background from the Vercel team:

```
For a bit more context and troubleshooting if you require, we initially realised that this needed to be added to turbo.json:
"globalPassThroughEnv": ["VERCEL"],

The reason being, svelte changes its behavior depending on the detected CI environment. It detects the CI environment via environment variable, and Turbo 2.0 changed the behaviour so that only environment variables included in turbo.json will be available to the app by default. So without the VERCEL env variable available to svelte, it didn’t know how to correctly format it’s output, and the public directory wasn’t generated.

Alternatively, you could swap from using sveltes adapter auto (the default, [docs](https://kit.svelte.dev/docs/adapter-auto) here) to adapter vercel ([docs](https://kit.svelte.dev/docs/adapter-vercel) here). This will eliminate the need for svelte to try and auto detect the CI environment.

However, this has all been updated in Turbo v2.0.8 and going forward so that you won't need to explicitly include these once updated. 
```
